theano/tensor/nnet/sigm.py

```diff
@@ -602,6 +602,7 @@ def local_exp_over_1_plus_exp(node):
             else:
                 # case: 1/(1+exp(x))
                 sigmoids.append(sigmoid(-t))
+            copy_stack_trace(node.outputs[0], sigmoids[-1])
```

Comment:
@lamblin Despite this change, several of the corresponding check_trace fail, the sigmoid outputs having no trace, I don't really understand why.